### PR TITLE
fix licence and remove unsupported Pair

### PR DESCRIPTION
**Pull request by tdurieux**

### CODE REVIEW

**Comment by GerardPaligot** [comment]
Pair isn't supported by what? Oo

**Comment by tdurieux** [comment]
By my JVM, 
Le 9 juil. 2016 9:05 AM, Gérard Paligot <notifications@github.com> a écrit :Pair isn't supported by what? Oo

**Comment by GerardPaligot** [comment]
PR not ready for merge.

**Comment by tdurieux** [comment]
can you rebase your branch, it is not up to date?
Travis fails on the javadoc

**Comment by GerardPaligot** [comment]
Commit cherry-picked.

**Comment by tdurieux** [comment]
perfect

**Comment by GerardPaligot** [comment]
Yes but Travis isn't happy for an obscure reason.

File: pom.xml [hunk 1]
```diff
@@ -165,7 +165,7 @@
   </scm>
 
   <properties>
-    <java.src.version>1.7</java.src.version>
+    <java.src.version>1.8</java.src.version>
```
Comment (tdurieux) [inline pom.xml]: @GerardPaligot In order to pass travis we have to use java 1.7 instead of java 1.8 :( my bad ...

**Pull request closed without merging**